javascript/README.md

```diff
@@ -93,6 +93,9 @@ Code snippets in Javascript demonstrating how to use various modules of the [Med
   * [duplicate_files.js](duplicate_files.js): list duplicates of the given files
 * [API:Filerepoinfo](https://www.mediawiki.org/wiki/API:Filerepoinfo)
   * [file_repo_info.js](file_repo_info.js): get information about file repositories
+* [API:Revisiondelete](https://www.mediawiki.org/wiki/API:Revisiondelete)
+  * [revision_delete.js](revision_delete.js): Hide all information about a certain revision ID. 
+  (The target, Sample Page, is unnecessary in this case.)
```

Comment:
The description is still too long :) I don't think we need to mention that `(The target, Sample Page, is unnecessary in this case)`. This information would be more appropriate in the documentation or in the comments :)